Assets/HoloToolkit/Utilities/README.md

```diff
@@ -14,6 +14,16 @@ A drop in direction indicator that stays in the users view at all times.
 #### HeadsUpDirectionIndicatorPointer.prefab
 A quad based pointer to be used with the HeadsUpDirectionIndicator prefab to create an out of box direction indicator.
 
+###[Scenes](Scenes)
+---
+
+####HeadsetAdjustment.unity
```

Comment:
@jessemcculloch, Pull Requests only require that two people review and approve before it can be merged.  I encourage you to test it out and review to move this forward.